src/seedu/addressbook/AddressBook.java

```diff
@@ -1146,12 +1159,12 @@ private static String getUsageInfoForExitCommand() {
     /**
      * Removes sign(p/, d/, etc) from parameter string
      *
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
+     * @param fullString  Parameter as a string
+     * @param prefix  Parameter sign to be removed
```

Comment:
Correct to update the comment. Note that you should remove the mention of "sign" in this comment too.